plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -246,10 +244,31 @@ def inspect(filter, options = {})
       ensure
         helper.remove_helper
       end
-      scope.extracted = false
 
+      scope.extracted = !!do_extract
```

Comment:
Is it? The double-bang is relatively common operator imo, I've seen it in several places before.